include/config_reader.h

```diff
@@ -0,0 +1,77 @@
+//
+// Created by Smirnov Grigory on 22.01.2022.
+//
+
+#ifndef CONFIG_READER_H
+#define CONFIG_READER_H
+
+#include <string>
+#include <iostream>
+#include <exception>
+#include <opencv2/core.hpp>
+
+class ConfigReader {
```

Comment:
if it is reader, then it should read config (fill structure with fields) from file and returns it, no need to store values in reader.

Reply:
ok.done